core-java-modules/core-java-io-apis-2/src/test/java/com/baeldung/inputstream/InputStreamTest.java

```diff
@@ -0,0 +1,58 @@
+package com.baeldung.inputstream;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+public class InputStreamTest {
```

Comment:
Please apply the Baeldung code formatter to all files in the PR, which you can find here:
https://github.com/Baeldung/dev-settings/tree/main/javaCodeStyle/intellij